scripts/src/report/verifier_report.py

```diff
@@ -84,6 +84,15 @@ def get_provider_delivery(report_data):
         pass
     return provider_delivery
 
+def get_chart_version(report_data):
```

Comment:
Are we using this function anywhere in the workflow?

Reply:
Good catch, that became unnecessary as my solution evolved. Removed.